desktop/src/com/frostwire/gui/bittorrent/CreateTorrentDialog.java

```diff
@@ -99,6 +96,24 @@
     private String _invalidTrackerURL;
     private JButton _buttonClose;
 
+    private JComboBox _pieceSizeComboBox;
+    private JLabel _labelPieceSize;
+    private int pieceSize;
+    private static final String[] PIECE_SIZE_OPTIONS = {
```

Comment:
I'm not sure if this is the proper way to do it. Since JComboBox constructor accepts just unidimensional iterables  i have to do it like this.  I was wondering if i could do it using a data structure Key, Value type, but after think it a little bit found that i will need a Array/Vector to populate selector options.